app/views/dashboard/show.html.erb

```diff
@@ -1,5 +1,36 @@
+<br><br><br><br>
 <% if current_user %>
-	<% current_user.attributes.each do |k, v| %>
-		<b><%= k %>:</b> <%= v %> <br>
-	<% end %>
-<% end %>
+	<div class="center-align"><h6> Welcome, <%= current_user.first_name.capitalize %>! Give us a hint on how you want to eat and we'll do the rest!</h6>
```

Comment:
Edit user dashboard to only show the user's name and then edit the styling for a more aesthetic feel.